SDL/Quintuplet.cu

```diff
@@ -428,6 +428,117 @@ __device__ bool SDL::runQuintupletDefaultAlgo(struct SDL::modules& modulesInGPU,
     pass = pass and tempPass;
     if(not pass) return pass;
 
+    short layer2_adjustment = 0;
+    if (modulesInGPU.layers[lowerModuleIndex1] == 1)
+    {
+        layer2_adjustment = 1; // get upper segment to be in second layer
+    }
+    bool is_endcap1 = (modulesInGPU.subdets[lowerModuleIndex1] == 4);
+    bool is_endcap2 = (modulesInGPU.subdets[lowerModuleIndex2] == 4);
+    bool is_endcap3 = (modulesInGPU.subdets[lowerModuleIndex3] == 4);
+    bool is_endcap4 = (modulesInGPU.subdets[lowerModuleIndex4] == 4);
+    bool is_endcap5 = (modulesInGPU.subdets[lowerModuleIndex5] == 4);
+    float x[38] = { 
+        log10(2.99792458e-3f*3.8f*innerRadius),                       // inner t3_pt
```

Comment:
We have a dedicated variable (`k2Rinv1GeVf`) to be used in such cases, defined in `Constants.cu`. Could you switch to that?

The same comment possibly applies to more cases below and for other constants, e.g. the magnetic field.

Reply:
Yes, `3.8f` should be the magnetic field, so we could at least change that. But, in any case, I think it would be good to understand what all the numerical factors are. Not sure who wrote that formula but @sgnoohc was taking a look at radius calculations at some point, so may know what's going on?

Reply:
Ok, I will make the replacement. The line comes from elsewhere in Quintuplet.cu:
```
float pt = (innerRadius+outerRadius)*3.8f*1.602f/(2*100*5.39f);
```